avm/src/lib.rs

```diff
@@ -134,6 +135,32 @@ pub fn uninstall_version(version: &Version) -> Result<()> {
     Ok(())
 }
 
+/// Read version from .vavm
+pub fn read_anchorversion_file() -> Result<Version> {
+    // get the root localisation
+    let path = env::current_dir().unwrap();
+    let dir_content = fs::read_dir(path).unwrap();
+    for entry in dir_content {
+        let entry = entry.unwrap();
+        let path_entry = entry.path();
+        if !path_entry.is_file() {
+            continue;
+        }
+
+        if path_entry
+            .file_name()
+            .unwrap()
+            .to_string_lossy() == ".anchorversion" {
+            let path_entry_content = fs::read_to_string(path_entry).unwrap();
+            let formated_content = path_entry_content.replace(|c: char| c.is_whitespace(), "");
+            let version = Version::parse(&formated_content).map_err(|e| anyhow!(e)).unwrap();
+            return Ok(version);
+        }
+    }
+
+    return Err(anyhow::format_err!(".anchorversion file not found. Please ensure the file exists in the current directory."));
```

Comment:
There is no need for getting the current directory, reading it and looping through the directory content because we already know what to look for.

```suggestion
    fs::read_to_string(".anchorversion")
        .map_err(|e| anyhow!(".anchorversion file not found: {e}"))
        .map(|content| Version::parse(content.trim()))?
        .map_err(|e| anyhow!("Unable to parse version: {e}"))
```